src/elements/Segment/Segments.js

```diff
@@ -0,0 +1,46 @@
+import React, {Children, Component, PropTypes} from 'react';
+import _ from 'lodash';
+import invariant from 'fbjs/lib/invariant';
+import classNames from 'classnames';
+import META from 'src/utils/Meta';
+import Segment from './Segment';
+
+export default class Segments extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+  };
+
+  componentDidMount() {
+    const {children} = this.props;
+
+    invariant(
+      !_.any( Children.map(children, child => child.type !== Segment) ),
+      'May only contain children of type Segment.',
+    );
+  }
+
+  static _meta = {
+    library: META.library.semanticUI,
+    name: 'Segments',
+    type: META.type.element,
+    parent: 'Segment'
+  };
```

Comment:
Very impressed the linter runs in a precommit hook. This was moved down to satisfy the `react/sort-comp` rule. Arguably we should update the linter rule so static assignments like this can be kept at the top for consistency between modules.